docs/index.rst

```diff
@@ -4,24 +4,20 @@
 CrateDB Npgsql Plug-In
 ======================
 
-A `.NET`_ plugin for `Npgsql`_ that provides backwards compatibility support
-for `CrateDB`_ versions 4.1.x and earlier. 
 
-.. WARNING::
+.. CAUTION::
 
-   CrateDB versions 4.2 and later work with `Npgsql`_ without the need for a plugin.
+    Please don't use this driver with recent versions of CrateDB.
 
-.. NOTE::
+    CrateDB versions prior to 4.2 needed a custom fork of the official `Npgsql`_
```

Comment:
I would leave a suggested edit here too, but the diff is broken up. this admonition should match the others though

Reply:
Addressed with 1d009e5a.